packages/stores/src/queries/concentrated-liquidity/tick-data-provider.ts

```diff
@@ -23,7 +23,7 @@ export class ConcentratedLiquidityPoolTickDataProvider
 
   constructor(
     protected readonly queryLiquiditiesNetInDirection: ObservableQueryLiquiditiesNetInDirection,
-    protected readonly nextTicksRampMultiplier = new Int(8),
+    protected readonly nextTicksRampMultiplier = new Int(9),
```

Comment:
Confirmed that this is the smallest possible to allow all tests to pass (and cover entire tick range)